src/webgpu/api/validation/buffer/mapping.spec.ts

```diff
@@ -184,12 +184,10 @@ g.test('mapAsync,state,mappingPending')
     // the mapping promise with an AbortError.
     t.shouldReject('AbortError', buffer.mapAsync(mapMode));
```

Comment:
Even with #2052 I don't think we're testing that this AbortError is deferred while the OperationError is immediate. Is it possible to replace calls to shouldReject with testMapAsyncCall so we can test that?